tests/core/Jalasoft.TeamUp.Resumes.Core.Tests/GetResumesByNameCoreTests.cs

```diff
@@ -0,0 +1,162 @@
+                },
+                new Resume()
+                {
+                    Id = 2,
+                    Title = "Marcelo Ruiz",
+                    Contact = new Contact
+                    {
+                        Address = "16 street",
+                        Email = "marcelo.ruiz@fundacion-jala.org",
+                        Phone = 123456789,
+                    },
+                    CreationDate = DateTime.Now,
+                    LastUpdate = DateTime.Now,
+                    Person = new Person
+                    {
+                        FirstName = "Marcelo",
+                        LastName = "Ruiz",
+                        Birthdate = DateTime.Now,
+                        Picture = "picture.jpg"
+                    },
+                    Summary = "I'm a great ux :-D",
+                    Skills = new List<Skill>()
+                    {
+                        new Skill
+                        {
+                            Id = 1,
+                            Name = "Figma"
+                        },
+                        new Skill
+                        {
+                            Id = 2,
+                            Name = "HTML"
+                        },
+                    }
+                }
+            };
+            return resumes;
+        }
+
+        [Fact]
+        public void GetResumesByName_NoItems_EmptyResult()
+        {
+            // Arrange
+            var stubEmptyResumeList = new List<Resume>();
+            this.mockRepository.Setup(repository => repository.GetByName("Gustavo")).Returns(stubEmptyResumeList);
+
+            // Act
+            var result = this.resumesService.GetByName("Gustavo");
+
+            // Assert
+            Assert.Empty(result);
+        }
+
+        [Fact]
+        public void GetResumesByName_ItemsExist_ResumesArray()
+        {
+            // Arrange
+            this.mockRepository.Setup(repository => repository.GetAll()).Returns(GetTestResumes().ToList());
+
+            // Act
+            var result = this.resumesService.GetResumes();
+
+            // Assert
+            Assert.Equal(2, result.Length);
+        }
+
+        [Fact]
+        public void GetResume_ValidId_SingleResume()
+        {
+            // Arrange
+            var stubResume = new Resume { Id = 1 };
+            this.mockRepository.Setup(repository => repository.GetById(1)).Returns(stubResume);
+
+            // Act
+            var result = this.resumesService.GetResume(1);
+
+            // Assert
+            Assert.NotNull(result);
+        }
+
+        [Fact]
+        public void GetResume_UnexistentID_ReturnNull()
+        {
+            Resume resp = null;
+
+            // Arrange
+            this.mockRepository.Setup(repository => repository.GetById(1)).Returns(resp);
+
+            // Assert
+            Assert.Null(this.resumesService.GetResume(1));
+        }
+
+        [Fact]
+        public void GetResumesByName_NotExit_NameResume()
```

Comment:
this name is not related to de implementation please modify to fit the actual code and follow the guidelines the team is using if any doubts reach to @niver-copa-jalafund  or @lilian-delgado-jalafund